api/python/quilt3/packages.py

```diff
@@ -1019,7 +1019,7 @@ def _set_commit_message(self, msg):
         if msg is not None and not isinstance(msg, str):
             raise ValueError(
                 f"The package commit message must be a string, but the message provided is an "
-                f"instance of {type(msg)}."
+                f"instance of {type(msg)}: {msg}."
```

Comment:
I'm not sure that makes much sense
imagine `msg` string representation is huge
doesn't seem that help readability

Reply:
reverting